apps/test/unit/templates/feedback/LevelFeedbackEntryTest.jsx

```diff
@@ -9,7 +9,7 @@ import {UnlocalizedTimeAgo as TimeAgo} from '@cdo/apps/templates/TimeAgo';
 import color from '@cdo/apps/util/color';
 import i18n from '@cdo/locale';
 
-const DEFAULT_FEEDBACK = {
+const NO_FEEDBACK = {
```

Comment:
Is this name change necessary? If so `NO_FEEDBACK` is confusing since it looks like it has feedback.

Reply:
I can switch it back to DEFAULT_FEEDBACK if that's clearer! Is there other wording that would help indicate not just the default, but the absence of feedback?

Reply:
I'm confused! Isn't there feedback here?

Also, I didn't quite understand the connection between this change and the overall update you're doing.

Reply:
> I'm confused! Isn't there feedback here?

no, prior to thumbs up or thumbs down being clicked, we are in a state of not having any feedback. LearningGoalAiEvaluation is the evaluation provided by AI. LearningGoalAiEvaluationFeedback is what's meant by "feedback" here, and represents the user's opinion of the AI evaluation, as indicated by thumbs up / thumbs down click, plus any additional info from the checkboxes + "other" textbox which show up when you click thumbs down. 

thus when the page loads and AI evaluation is present but the user has not yet clicked and thumbs up / down button, we are in a state of "no feedback". not sure if that helps clear things up? 

> Also, I didn't quite understand the connection between this change and the overall update you're doing.

we had local definitions of `thumbsupval`, `thumbsdownval`, and `-1` scattered throughout the code, so I did a refactoring step to centralize these all as constants. Please see links in the PR description to see code, test and refactoring diffs separately.

Reply:
I understand the changes and refactoring you have done to the AI feedback components. I'm not sure about this particular change to the LevelFeedbackEntry component test file. This component seemed somewhat distant from the AI assessment logic... although we _do kinda_ touch the keep working data.

I was presuming that this was student progress feedback, and from my understanding it has "`Great work`" as the student feedback, so it was a little hard to understand why this was `NO_FEEDBACK` and felt like maybe an accidental find/replace.

Reply:
oh jeez, I was totally unaware that i had touched an unrelated file. you are correct, I will revert!

Reply:
No worries! Easy peasy lemon squeezy.